src/library_sdl.js

```diff
@@ -2460,16 +2477,16 @@ var LibrarySDL = {
   },
 
   SDL_PauseAudio: function(pauseOn) {
-    if (!SDL.audio) {
+    if (!SDL.audio || (SDL.audio.paused && pauseOn) || (!SDL.audio.paused && !pauseOn)) {
       return;
     }
     if (pauseOn) {
-      if (SDL.audio.timer !== undefined) {
```

Comment:
Was there a specific reason for removing the `if (SDL.audio.timer !== undefined)` condition here?